GVRf/Framework/src/org/gearvrf/GVRScene.java

```diff
@@ -212,8 +213,21 @@ void updateStats() {
 
             mStatsConsole.writeLine("Draw Calls: %d", numberDrawCalls);
             mStatsConsole.writeLine(" Triangles: %d", numberTriangles);
+            
+            if(mStatMessage!="")
+            	mStatsConsole.writeLine("%s", mStatMessage);
```

Comment:
Have you tested this? It doesn't reset the field; you will get the same message, over and over.